keywords/couchbaseserver.py

```diff
@@ -247,6 +207,50 @@ def wait_for_ready_state(self):
             # All nodes are heathy if it made it to here
             break
 
+    def _create_internal_rbac_bucket_user(self, bucketname):
```

Comment:
I see. I may disable that pylint warning. Some people use _prefix to denote an internal method to the class, some people opt against it due to it not being pure pythonic. I think it is useful